scripts/build/docker/Dockerfile.template

```diff
@@ -43,6 +43,12 @@ RUN apt-get update \
     zip \
     rpm
 
+# Workaround: Install a newer version of `tar` to make fpm in electron-builder work again
+RUN echo "deb http://ftp.de.debian.org/debian wheezy main contrib non-free" >> /etc/apt/sources.list
```

Comment:
I guess we don't need the `non-free`. We can also try to remove `contrib`

Reply:
Yeah, `libacl1` is on `main`

Reply:
Done